src/app.js

```diff
@@ -1 +1,5 @@
 'use strict';
+
+const { randomNumberGenerator } = require('./randomNumberGenerator');
+
+randomNumberGenerator();
```

Comment:
`randomNumberGenerator` should not be the implementation of entire game. This violates [sepation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns). 

Random number generator should just return a number, no logging, nothing else.
calculateBullsAndCows should not output anything to screen, it should just return something like `{bulls: 0, cows: 2}`.
inputOutput should just export functions for logging and question, no game logic.

And then you need to call all those functions in your main `app.js` to implement a game.